src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -322,8 +324,12 @@ std::vector<ov::Tensor> SubgraphBaseTest::get_plugin_outputs() {
 }
 
 void SubgraphBaseTest::validate() {
-    auto expectedOutputs = calculate_refs();
-    const auto& actualOutputs = get_plugin_outputs();
+    std::vector<ov::Tensor> expectedOutputs, actualOutputs;
+
+    std::thread t_device([&]{ actualOutputs = get_plugin_outputs(); });
```

Comment:
Is it possible to have an option to disable async execution using
an environment variable for example?
I think it is highly likely that this async execution might make it harder to debug / troubleshoot issues.

Reply:
done